docs/topics/what-is-the-toolkit.adoc

```diff
@@ -23,6 +23,10 @@ The {ProductName} ({ProductShortName}) is an extensible and customizable rule-ba
 * Containerizing applications and making them cloud-ready
 * Migrating from Java Spring Boot to Quarkus
 * Updating from Oracle JDK to OpenJDK
+* Upgrading from OpenJDK 8 to OpenJDK 11
+* Upgrading from OpenJDK11 to OpenJDK 17
+* Migrating EAP Java applicatons to Azure
+* Migrating Spring Boot Java applications to Azure
 
 For more information about use cases and migration paths, see the link:https://developers.redhat.com/products/mta/use-cases[MTA for developers] web page.

```

Comment:
mta to mtr

Reply:
line 44 also

Reply:
@stillalearner But the link leads to an MTA page - is it correct?